mlir/include/mlir/Transforms/DialectConversion.h

```diff
@@ -688,8 +688,9 @@ class OpConversionPattern : public ConversionPattern {
     return matchAndRewrite(op, OpAdaptor(oneToOneOperands, adaptor), rewriter);
   }
 
-private:
+protected:
```

Comment:
The `using` here is only to silence warnings. I think it won't be a problem to turn it into `protected`.